module/create_connection.py

```diff
@@ -9,3 +11,29 @@ def create_connection(db_file: str) -> sqlite3.Connection:
     except Error as e:
         print(str(e))
     return conn
+
+
+def connect_and_execute(context: CallbackContext, chat_id: int, query: str, params: tuple, operation: str) -> Union[int, list, None]:
+    conn = create_connection(DB_PATH)
+    if not conn:
+        context.bot.send_message(chat_id, DB_ERROR)
+        return
+    
+    cur = conn.cursor()
+    cur.execute(query, params)
+    if operation == INSERT:
+        conn.commit()
+        last_row_id = cur.lastrowid
+        conn.close()
+        return last_row_id
+    
+    elif operation == DELETE:
+        conn.commit()
+        conn.close()
+        return
+
+    elif operation == SELECT:
+        rows = cur.fetchall()
+        conn.close()
+        return rows
```

Comment:
the `elif` can become `if`
if you use `return` in all `ifs` you can convert `elif` into `if`

Reply:
moreover, here you should use match for `operation` https://learnpython.com/blog/python-match-case-statement/